docs/sources/packages_api/data/featuretoggles.md

```diff
@@ -30,6 +30,7 @@ import { FeatureToggles } from '@grafana/data';
 |  [meta](#meta-property) | <code>boolean</code> |  |
 |  [newEdit](#newedit-property) | <code>boolean</code> |  |
 |  [reportGrid](#reportgrid-property) | <code>boolean</code> |  |
+|  [standaloneAlerts](#standalonealerts-property) | <code>boolean</code> |  |

```

Comment:
Have you updated this manually? Think this file is generated after a stable release.

Reply:
Not manually. I have generated it by running `scripts/ci-reference-docs-build.sh` but I'm also unsure if it's required at this phase. Let's see what the frontend people think otherwise I'll revert the [commit](https://github.com/grafana/grafana/pull/25984/commits/f6ca7e7ec96ac85b28622863ee8927423ea82a0d)

Reply:
@papagian no need for comiting these changes. We regenerate packages docs during the release anyways.